jekyll/dataguide_hazards/var.md

```diff
@@ -16,13 +16,54 @@ Using hazard exposure data on the frequency of different event intensities, we t
 
 The curves provided by the Army Corps were designed specifically for the United States.  In order to adapt them for global use cases, we apply regional calibrations using data from the [CLIMADA](https://nhess.copernicus.org/articles/21/393/2021/) risk analysis framework.  These calibrations account for large-scale differences in building standards around the world.  For example, these calibrations allow us to more accurately describe the impact of 1m of flooding in Japan (where building standards are very high) and other places of the world where standards may be lower.
 
 ### Cyclones
-To calculate VaR for cyclones, we start with wind speed exposure data in the form of <i>annual exceedance probabilities</i> (also known as Return Periods).  This data essentially describes the annual likelihood of a given wind speed.  We combine this data with regionally specific vulnerability curves from the [CLIMADA](https://nhess.copernicus.org/articles/21/393/2021/) risk analysis framework.  These functions (also known as <i>impact functions</i> or <i>vulnerability curves</i>) describe the expected damage to a given structure type for a given wind speed.  For example, a wind speed of 60 meters per second may damage 50% of a structure's total value, for the typical single family wooden home.
+To calculate financial risk for cyclones, we start with wind speed exposure data in the form of <i>annual exceedance probabilities</i> (also known as Return Periods).  This data essentially describes the annual likelihood of a given wind speed.  We combine this data with regionally specific vulnerability curves from the [CLIMADA](https://nhess.copernicus.org/articles/21/393/2021/) risk analysis framework.  These functions (also known as <i>impact functions</i> or <i>vulnerability curves</i>) describe the expected damage to a given structure type for a given wind speed.  For example, a wind speed of 60 meters per second may damage 50% of a structure's total value, for the typical single family wooden home.
 
 ### Wildfires
 For wildfire, we take a different approach to measuring damage.  In the event of exposure to wildfire, we assume that 75% of an asset is damaged.  This number is derived from research on historic impacts from wildfires in California.  
 
+## Asset Type Specific Damages
+In order to more accurately calculate financial risk, it is possible to upload asset types, via the `label:asset_type` column.  This allows us to apply specialized impact functions based on the uploaded asset type, accounting for industry-specific characteristics.
```

Comment:
I believe it is actually just `label:type`.  By the time the data gets to the geoexplorer operator, the column needs to be `type`.  So unless at some point we clean off `asset_` as a column prefix, I think it should be `label:type`